invenio_deposit/api.py

```diff
@@ -308,3 +308,13 @@ def sort_by(*args, **kwargs):
             files_.sort_by = sort_by
 
         return files_
+
+    def get_file_by_id(self, file_id):
```

Comment:
how it works? Will I leave the file endpoint with the key and then zenodo make some tricks to use the file_id?